src/PhpTemplateRenderer.php

```diff
@@ -4,21 +4,29 @@
 
 namespace Yiisoft\View;
 
+use Yiisoft\View\Exception\ContentCantBeFetched;
+
 class PhpTemplateRenderer implements TemplateRendererInterface
 {
     public function render(View $view, string $template, array $params): string
     {
-        $renderer = function () {
-            extract(func_get_arg(1), EXTR_OVERWRITE);
-            require func_get_arg(0);
+        $renderer = function (string $template, array $params) {
```

Comment:
That's not a good change. It pollutes the scope of the view rendering and user can't use `template` in as variable name in the views anymore.

Reply:
Actually, that's not polluting the scope due to EXTR_OVERWRITE option. But variables $template and $params is always defined